src/components/molecules/VenueChat/VenueChat.tsx

```diff
@@ -27,6 +32,16 @@ interface ChatOutDataType {
 
 const VenueChat: FC = () => {
   const venueId = useVenueId();
+  const usersById = useSelector(venueChatUsersSelector) ?? {};
+  const venueChatUsersQuery = useMemo<ReduxFirestoreQuerySetting>(
```

Comment:
Should this be `SparkleRFQConfig` instead of `ReduxFirestoreQuerySetting`?

Reply:
The `useMemo` probably isn't needed here as `useFirestoreConnect` does a deep compare on the query it's passed anyway.